src/managed/CommonManaged.props

```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
     <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../..'))/</RepoRoot>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>$(VersionPrefix)</VersionPrefix>
```

Comment:
Also, I believe this change is going to cause the suffix to be appended twice.  Just keep this hard-coded for now.  The referenced bug will fix this for real.